application/classes/Ushahidi/Formatter/Post/GeoJSON.php

```diff
@@ -32,6 +32,7 @@ public function __invoke($entity)
 						'properties' => [
 							'title' => $entity->title,
 							'description' => $entity->content,
+							'color' => $entity->color,
```

Comment:
Can we use `marker-color` instead? (as in comments below) Probably pick a sane default for `marker-symbol` too